src/main/java/org/example/InputOutput.java

```diff
@@ -8,20 +8,23 @@
 
 public class InputOutput {
 
-    public String getUserInput(BufferedReader bufferedReader) throws IOException {
-        return bufferedReader.readLine();
+    public String getUserInput() throws IOException {
```

Comment:
Is this method used somewhere? Do we still need it?

Reply:
Agreed, no longer needed. Removed from the application.